components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -611,8 +611,8 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
                 c_Visbeck = sqrt(sumN2*ltsum)
                 sumRi = sumRi / (ltsum + 1.0E-11_RKIND)
 
-                eddyLength(iEdge) = min(c_Visbeck/(1.0E-15_RKIND + abs(fEdge(iEdge))), &
```

Comment:
@vanroekel this array was created in the original Visbeck addition. Both `eddyLength` and `eddyTime` are used only in these three lines within the `iEdge` loop, so these arrays could both easily be replaced by scalars. The only reason to keep them is for writing them out. Do you have a preference if these are arrays or scalars?

Reply:
@mark-petersen no real preference on scalar v array.  I had them as arrays for diagnostics but I don't think we need them for the duration.